core/unit_test/CMakeLists.txt

```diff
@@ -546,6 +546,7 @@ IF(KOKKOS_ENABLE_OPENACC AND KOKKOS_CXX_COMPILER_ID STREQUAL NVHPC)
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewAPI_f.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewMapping_b.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewResize.cpp
+    ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_BitManipulationBuiltins.cpp
```

Comment:
How is that change related?

Reply:
Unrelated to this PR; just added it because NVHPC fails.

Reply:
Undo and propose in another PR as necessary

Reply:
Removed; now the OpenACC unit test will fail.

Reply:
This failed unit test problem is handled by issue #6088.